packages/react-devtools-shared/src/__tests__/__snapshots__/profilingCache-test.js.snap

```diff
@@ -3279,8 +3283,12 @@ Object {
           1,
           0,
           2,
-          1,
+          5,
           4,
+          11,
+          10,
+          8,
+          7,
```

Comment:
I didn't anticipate this change. (Some change, maybe, but this specific change seems odd to me.)